src/main/java/dev/galacticraft/mod/block/GalacticraftBlock.java

```diff
@@ -86,6 +86,10 @@
             .allowsMovement().destroyedByPiston().burnable().lightPassesThrough().notSolid().replaceable().liquid().build())
             .strength(50.0F, 50.0F).dropsNothing()), Constant.Block.FUEL);
 
+    public static final dev.galacticraft.mod.api.block.FluidBlock BACTERIAL_SLUDGE = registerBlockWithoutItem(new BacterialSludgeBlock(GalacticraftFluid.BACTERIAL_SLUDGE, FabricBlockSettings.of(new FabricMaterialBuilder(MaterialColor.GREEN)
```

Comment:
`dev.galacticraft.mod.api.block.FluidBlock` is already imported on L26, this can be shortened to just the class name.

Reply:
We should probably just AW FluidBlock's constructor rather than having this class

Reply:
Yeah I think there's still a lot of things from before access wideners